packages/app/src/sandbox/eval/transpilers/babel/convert-esmodule/__snapshots__/index.test.ts.snap

```diff
@@ -707,12 +726,128 @@ var $csb__2 = require(\\"2\\");
 "
 `;
 
+exports[`convert-esmodule keeps star exports after default export order 1`] = `
+"\\"use strict\\";
+exports.a = exports.WithSearch = exports.withSearch = void  0;
+Object.defineProperty(exports, \\"__esModule\\", {
+  value: true
+});
+var $csb__withSearch = $_csb__interopRequireDefault(require(\\"./withSearch\\"));
+Object.defineProperty(exports, \\"withSearch\\", {
+  enumerable: true,
+  configurable: true,
+  get: function $csbGet() {
+    return $csb__withSearch.default;
+  }
+});
+var $csb__WithSearch = $_csb__interopRequireDefault(require(\\"./WithSearch\\"));
+Object.defineProperty(exports, \\"WithSearch\\", {
+  enumerable: true,
+  configurable: true,
+  get: function $csbGet() {
+    return $csb__WithSearch.default;
+  }
+});
+var $csb__test = require(\\"./test\\");
+Object.defineProperty(exports, \\"a\\", {
+  enumerable: true,
+  configurable: true,
+  get: function $csbGet() {
+    return $csb__test.a;
+  }
+});
+var $csb__containers = require(\\"./containers\\");
+function $_csb__interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : {
+    default: obj
+  };
+}
+Object.keys($csb__containers).forEach(function (key) {
+  if (key === \\"default\\" || key === \\"__esModule\\") return;
+  if (Object.prototype.hasOwnProperty.call(exports, key)) return;
+  Object.defineProperty(exports, key, {
+    enumerable: true,
+    configurable: true,
+    get: function $csbGet() {
+      return $csb__containers[key];
+    }
+  });
+});
+"
+`;
+
 exports[`convert-esmodule parses and writes chars with linebreaks 1`] = `
 "\\"use strict\\";
 var WS_CHARS = 'u2000-\\\\u200a\\\\u2028\\\\u2029\\\\u202f\\\\u205f\\\\u3000\\\\ufeff';
 "
 `;
 
+exports[`convert-esmodule predefines possible exports 1`] = `
+"\\"use strict\\";
+exports.default = exports.l = exports.j = exports.bah = exports.f = exports.e1 = exports.d = exports.c = exports.b = exports.a = void  0;
```

Comment:
Nit: extra space in `void  0`.

Reply:
Yeah that one is nagging me too. It's defined in the generator of `astring`, so I'm not 100% sure if I want to touch that...

Reply:
Ah, nvm, I thought it comes from your test code. 🙂